ops/opset13/greater.go

```diff
@@ -0,0 +1,66 @@
+package opset13
+
+import (
+	"github.com/advancedclimatesystems/gonnx/onnx"
+	"github.com/advancedclimatesystems/gonnx/ops"
+	"gorgonia.org/tensor"
+)
+
+var (
+	MinGreaterInputs = 2
+	MaxGreaterInputs = 2
+)
+
+// Greater represents the ONNX greater operator.
+type Greater struct{}
+
+// newGreater creates a new greater operator.
+func newGreater() ops.Operator {
+	return &Greater{}
+}
+
+// Init initializes the greater operator.
+func (g *Greater) Init(_ []*onnx.AttributeProto) error {
+	return nil
+}
+
+// Apply applies the greater operator.
+func (g *Greater) Apply(inputs []tensor.Tensor) ([]tensor.Tensor, error) {
+	in1, in2, err := ops.MultidirectionalBroadcast(inputs[0], inputs[1])
```

Comment:
The implementation of these comparison operators are all very similar. Maybe we could get rid of some boilerplate if you implemented something like: 

```go
type CompOperator func(a, b tensor.Tensor) (tensor.Tensor, err) 

func applyComparisonOperator(inputs []tensor.Tensor, op CompOperator) ([]tensor.Tensor, error) {
    in1, in2, err := ops.MultidirectionalBroadcast(inputs[0], inputs[1])
    if err != nil {
         return nil ,err 
    } 

    out, err := op(inputs[0], input[1])
    if err != nil {
        return nil, err
    }

    return []tensor.Tensor{out}, nil
}


// Then in the actual operator you could do a simple oneliner like:
return applyComparisonOperator(inputs, tensor.Gt)

```